Cargo.toml

```diff
@@ -23,3 +23,4 @@ crate-type = ["cdylib", "rlib"]
 tracing = {version = "0.1", features = ["attributes"]}
 tracing-subscriber = {version = "0.2", features = ["registry"]}
 wasm-bindgen = {version = "0.2"}
+chrono = { version = "^0.4", features = ["wasmbind"] }
```

Comment:
I've seen `chrono` often put behind a feature flag. Is that possibly to allow avoiding extra dependencies?

We might consider adding a feature-flag like that here as a follow-up to this PR.

Reply:
I am not sure about that, `chrono` was once incompatible with wasm, which had already caused me issues, but is fixed now apparently. It may probably still be useful to have a feature flag for time logging.

Reply:
Being able to avoid extra dependencies is always a good thing, potentially leading to shorter compile times and less network bandwidth used for the build. On the other hand, every feature carries a complexity cost, and unfortunately features aren't that discoverable yet. As chrono is fairly small and very compatible (depending on features, it can even be `no_std`), I don't think we'll need to put it behind a feature. I don't understand the usage sufficiently to gauge if we should at least make `wasmbind` a default feature otherwise.